test/integration/builder/inserts.js

```diff
@@ -1302,5 +1302,17 @@ module.exports = function(knex) {
           });
       });
     });
+
+    describe('#3553 insert with bigint', function() {
+      if (typeof BigInt === 'undefined') return;
+
+      it('should allow insert with BigInt', async function() {
+        await knex('accounts').insert([{ id: BigInt(12) }]);
+      });
+      it('should allow insert array with BigInt', async function() {
+        if (knex.client.driverName !== 'pg') return true;
```

Comment:
Might want to do `return this.skip()` for now.

(Eventually, we should figure out a way to restructure the test cases so that these skipped scenarios get omitted.  But, marking them as `pending` will at least remind us to revisit this later)